java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture3/tasks/_1/Number1.java

```diff
@@ -0,0 +1,204 @@
+                piano.getName();
+                musicians.setInstrument(piano);
+                orchestra.addMusician(musicians);
+            }
+            case 2 -> {
+                Instruments violin = new Violin("VI VI VI", "Violin");
+                System.out.print("Instrument is: ");
+                violin.getName();
+                musicians.setInstrument(violin);
+                orchestra.addMusician(musicians);
+            }
+            case 3 -> {
+                Instruments guitar = new Guitar("DSUCH DSUCH DSUCH", "Violin");
+                System.out.print("Instrument is: ");
+                guitar.getName();
+                musicians.setInstrument(guitar);
+                orchestra.addMusician(musicians);
+            }
+            default -> throw new IllegalStateException("Unexpected value: " + scanner.nextInt());
+        }
+    }
+}
+
+class Orchestra {
+
+    private ArrayList<Musicians> list;
+
+    public Orchestra(ArrayList<Musicians> list) {
+        this.list = list;
+    }
+
+    public void play() {
+        for (Musicians musicians : list) {
+            musicians.instrument.getSound();
+        }
+    }
+
+    public void addMusician(Musicians musicians) {
+        this.list.add(musicians);
+    }
+
+    public void removeMusician(int integer) {
+        var variable = this.list.remove(integer);
+        System.out.println("Removed" + variable);
+    }
+
+    public void amountOfMusicians() {
+        System.out.println(list.size());
+    }
+
+    @Override
+    public String toString() {
+        return "Orchestra{" + "\n" + list +
+                '}';
+    }
+}
+
+class Musicians {
+
+    Instruments instrument;
+    private static int id = 0;
+    private int incrementId = ++id;
+
+    private String name;
+
+    @Override
+    public String toString() {
+        return "Musician: " + "[id" + " " + incrementId + ", name of a musician = " + name + ']' + "\n" +
+                '[' + instrument + ']' + "\n";
+    }
+
+    public Musicians(String name) {
+        this.name = name;
+    }
+
+    public void getName() {
+        System.out.println(name);
+    }
+
+    public void setInstrument(Instruments instrument) {
+        this.instrument = instrument;
+    }
+
+    public void getInstrument() {
+        System.out.println(instrument);
+    }
+
+    public static int getId() {
+        return id;
+    }
+}
+
+class Instruments {
+    private final String sound;
+    private final String name;
+
+    public Instruments(String sound, String name) {
+        this.sound = sound;
+        this.name = name;
+    }
+
+    public void getName() {
+        System.out.println(name);
+    }
+
+    public void getSound() {
+        System.out.println(sound);
+    }
+
+    @Override
+    public String toString() {
+        return "His/Her instrument is " + name + ", " +
+                "it makes sound: " + sound;
+    }
+}
+
+class Piano extends Instruments {
```

Comment:
Try to define new classes for objects only if they differ. As for now, there is no difference between classes `Guitar`, `Piano` and `Violin` -  you can simply use a single class `MusicalInstrument`.

```
Each instrument has a name and a method to print a unique sound of the instrument.
```

The initial idea was that sound and name are not passed to the constructor - each class defines them.